oap-server/server-receiver-plugin/receiver-proto/src/main/proto/satellite/envoy/accesslog/v2/als.proto

```diff
@@ -0,0 +1,43 @@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+syntax = "proto3";
+
+package satellite.envoy.accesslog.v2;
+
+import "envoy/service/accesslog/v3/als.proto";
+
+option java_package = "org.apache.skywalking.satellite.envoy.accesslog.v2";
+option java_outer_classname = "SatelliteAlsProto";
+option java_multiple_files = true;
+option java_generic_services = true;
+
+// [#protodoc-title: Satellite gRPC Access Log Service (ALS)]
+
+// The new Envoy ALS protocol, work for satellite transmit the ALS message to oap.
+service SatelliteAccessLogService {
+  // Use the same parameters to transmit access log messages.
+  // The only difference is that the identity information (StreamAccessLogsMessage#identity) may occur on each message.
+  // Rely on the streaming messages are orderly, so there will be no problems with message processing.
+  // Therefore, when the satellite transmits the ALS message, it does not need to open, send and close the stream for each different identity (envoy).
+  // As a result, unnecessary streaming operation requests could be reduced, and the satellite becomes more stable when the satellite sends requests to the upstream.
+  // Especially when the number of envoys increases, the optimization becomes more obvious.
+  rpc StreamAccessLogs(stream .envoy.service.accesslog.v3.StreamAccessLogsMessage) returns (.envoy.service.accesslog.v3.StreamAccessLogsResponse) {
```

Comment:
This should be `v2`?

Reply:
Currently, the OAP ALS handler has combined the `v2` and `v3` version message and response, and they do not contain conflict. So I think we could just keep it, all they use the `v3` version.

Reply:
For messages they are all v3, but for gRPC service they are different versions, if you want to keep in v3 service between Satellite and OAP, then I don't think you need to copy `v3/als.proto` to `v2/als.proto`, they are exactly the same.

Reply:
v3 should be correct. But this reminds me, do we really need v2? Satellite should be 100% good to forward v2 ALS to the v3 endpoint of OAP, right? We could add less codes in OAP for now. v3 is the only main stream.

Reply:
@kezhenxu94 That is what you did :) Maybe you forget it. v2 gRPC service with v3 parameter and return values.

https://github.com/apache/skywalking/blob/842b5d927e9c196e14a4dc3f73812d2c4adfebf4/oap-server/server-receiver-plugin/receiver-proto/src/main/proto/envoy/service/accesslog/v2/als.proto

Reply:
Anyway, I still prefer to using v3 only.

Reply:
I know it, I remember it...

But what @mrproliu did is provide 2 exactly the same service with same message, what I did is 2 different service with same protocol buffer message, different Envoy version send via different service version.

**HIGHLIGHT**: service version and message version

Reply:
The only difference is `package name`, which is the ALS difference. Like I mentioned, this is not important at all. 
Use only `v3/als/proto` please @mrproliu

Reply:
been removed v2 version protocol.